### PR TITLE
Update Installation.md

### PR DESCRIPTION
**Pull request by Stalker71**
Update the documentation to fix the permission rights issue

### CODE REVIEW

**Comment by murrant** [comment]
A more secure fix is to give the librenms group write access on storage and bootstrap/cache and add the webserver user to the librenms group.

**Comment by f0o** [comment]
I agree with @murrant

**Comment by Stalker71** [comment]
Totally agree with you and made the change:

sudo chown -R www-data:www-data storage

File: docs/Install/Installation.md [hunk 1]
```diff
@@ -19,18 +19,37 @@ Requirements
...
+
+```bash
 composer install
 php artisan key:generate
 ```
 
+Make sure webserver can write logs in storage directory
+
+```bash
+sudo chown -R www-data:www-data storage
```
Comment (murrant) [inline docs/Install/Installation.md]: I would rather see this be:
chmod -R g+w storage logs
usermod -a -G librenms www-data

And up near the section where you do the other chown.

File: docs/Install/Installation.md [hunk 2]
```diff
@@ -19,18 +19,37 @@ Requirements
...
+```
+
+```bash
 cp .env.example .env
 ```
 
 Please now edit the .env file and set your options.
 
 ```bash
+replace Travis DB configuration with the one used by Librenms (have a look at the config.php file in /opt/librenms directory)
```
Comment (murrant) [inline docs/Install/Installation.md]: Change "replace Travis DB" to "Replace the database" and move this line out of the code block.

**Comment by murrant** [comment]
Thanks @Stalker71 .  Can you make that last change I noted, and I'll merge.

**Comment by Stalker71** [comment]
Sorry @murrant I didn't catch your note. Indeed thank you for your support!

**Pull request merged**